lib/modules/manager/dockerfile/extract.ts

```diff
@@ -168,24 +168,26 @@ export function getDep(
   specifyReplaceString = true,
   registryAliases?: Record<string, string>,
 ): PackageDependency {
-  if (!is.string(currentFrom) || is.emptyStringOrWhitespace(currentFrom)) {
+  if (
+    !is.string(currentFrom) ||
+    !is.nonEmptyStringAndNotWhitespace(currentFrom)
```

Comment:
I guess the `is.string` is because of the type assertion of `nonEmptyStringAndNotWhitespace` does not narrow this to `string`?

Reply:
yes, before it narrowed the type here to string, so after this condition the type was never. 🤷‍♂️